### ISSUE TITLE
I miss a wrong in 'scheduler.py'

### ISSUE THREAD

**Issue by SkevyHoo** [FanChiMao/HWMNet#4]
Sry,when I attemped to use your code,in scheduler.py,there's a wrong Cannot find reference '_LRScheduler' in 'lr_scheduler.pyi'of `from torch.optim.lr_scheduler import _LRScheduler`.So,how can I solve it ?May you give me a requeirment.txt about your environment? Thanks！

**Comment by FanChiMao** [comment]
You can refer https://github.com/FanChiMao/HWMNet/issues/1#issuecomment-1066604302, I think it's a environment problem maybe. 🤔
